src/elements/content-sidebar/MetadataSidebarRedesign.scss

```diff
@@ -1,5 +1,19 @@
-@import '../common/variables';
+@import '~@box/blueprint-web-assets/tokens/tokens.scss';
 
 .bcs-MetadataSidebarRedesign {
-    padding-inline: 10px;
+    background-color: $gray-02;
+    border-left: 1px solid $gray-10;
+
+    &-header {
```

Comment:
historically we've written the full name instead of `&` for searchability

Reply:
changed